src/Temporalio/Workflows/Workflow.cs

```diff
@@ -1080,14 +1082,11 @@ public static Guid NewGuid()
         /// <param name="options">Workflow options.</param>
         /// <returns>The child workflow handle once started.</returns>
         /// <remarks>
-        /// The task can throw a <see cref="Exceptions.WorkflowAlreadyStartedException" /> if an ID
-        /// is given in the options but it is already running.
-        /// </remarks>
-        /// <remarks>
         /// Using an already-cancelled token may give a different exception than cancelling after
         /// started. Use <see cref="Exceptions.TemporalException.IsCanceledException(Exception)" />
         /// to check if it's a cancellation either way.
         /// </remarks>
+        /// <exception cref="Exceptions.WorkflowAlreadyStartedException">Throw if an ID is given in the options, but it is already running.</exception>
```

Comment:
_Technically_ this doesn't throw this exception, the task does. I know .NET docs often only use this tag if calling the method sans `await` would throw the exception which isn't the case here. For example, see how `Task.WhenAny` or `Task.WhenAll` document their exceptions. So if you had:

```csharp
try
{
    var task = Workflow.StartChildWorkflowAsync(...);
}
catch (WorkflowAlreadyStartedException e)
{
    // ...
}
```
You would not catch it but this documentation now makes you think you would. This kind of code is common in workflows:

```csharp
var handleTasks = Enumerable.Range(0, 10).Select(i => Workflow.StartChildWorkflowAsync(...)).ToList();
var handles = await Task.WhenAll(handleTasks);
```

Sometimes we use this tag in other SDK docs even if it only occurs on await because it's inconsequential to differentiate, but it's fairly important for deterministic coroutine reasons that we are accurate with workflow docs.

Reply:
Got your point, so we can still it phrase it like `If awaited, can be thrown...`. What do you think?

Reply:
Sure, but that's basically what the paragraph above said. Not sure we want to set the precedent of using `<exception` tags in `Workflow` docs for cases where the exception is thrown in a potentially-separate coroutine.

Reply:
Hmm, that is interesting. I thought `exception` is the thing that Microsoft does, do you have some sample link of what they do? Thank you

Reply:
Well, that makes sense.I will adjust it.